notebooks/paper-notebook.ipynb

```diff
@@ -265,7 +265,7 @@
     "```\n",
     "where the first three optional arguments specify a strategy for iterating across the batch: Fortran's loop-parallel `do concurrent` construct, OpenMP multithreading, or an `elemental` procedure that operates on whole `tensor_t` arrays or array slices in one `infer` invocation.  If none of the first three optional arguments exists on the command line, then all three execute.  The fourth optional argument decides whether to additionally perform inference using double precision. The final optional argument determines the number of times each strategy will execute.  See {cite}`rouson2025automatically` for a discussion of the performance of each of these approaches.\n",
     "\n",
-    "Before running `concurrent-inferences`, download the pretrained aerosol model file [model.json](./assets/model.json) and save it to the root directory of your Fiats clone. \n",
+    "Before running `concurrent-inferences`, download the pretrained aerosol model file [model.json](https://github.com/ucar-sea/SEA-ISS-2025-Cloud-microphysics-training/blob/main/assets/model.json) and save it to the root directory of your Fiats clone. \n",
```

Comment:
Similar to the point made elsewhere by @rouson, I believe the change above converts a robust repo-relative link into a fragile absolute link.

This hunk should probably be reverted unless there's a strong reason for doing this?

Reply:
Unfortunately, this wasn't rendering properly as a link in the web version without the absolute link.  I suspect this has something to do with the format and it not being a source file, but don't know for sure.  I also would have assumed it should work.

I was a bit less concerned with this particular link since it references the source repository, but understand the concern.